lib/src/_isolates_io.dart

```diff
@@ -32,6 +34,7 @@ class PostgrestIsolate {
 
   Future<dynamic> decode(String json) async {
     if (!_createdIsolate.isCompleted) {
+      if (!_startedInit) init();
```

Comment:
I think we need to await here, right?
```suggestion
      if (!_startedInit) await init();
```

Reply:
We can, but don't have to, because the awaited completer below handles that as well.

Reply:
Ah, I see!